### PR TITLE
Nicholas contact

### PR DESCRIPTION
**Pull request by NR229**
Basic changes for contact page.

### CODE REVIEW

**Review by MengtingCao — APPROVED** [review]
Looks good

**Review by jswood23 — CHANGES_REQUESTED** [review]
Do you know what's wrong with the deployment that the image is not rendering?

File: app/views/external/contact.html.erb [hunk 1]
```diff
@@ -1,2 +1,11 @@
...
-<p>Contact page stuff goes here</p>
\ No newline at end of file
+<div class="content" style="display: flex;">
+    <div class="container" style="margin: auto; display: flex; flex-direction: column">
+        <h1 style="margin: auto;">Contacts</h1>
+        <img src="/assets/default.jpg" alt="profile" style="width: 30vw; margin: auto; max-width: 250px;">
+        <p style="margin: auto;">Below you can find our email and social links, be sure to follow MCHSO on Facebook!</p>
+        <ul style="margin: auto">
+            <li><a href="mailto:hannahbartosh@tamu.edu">Email</a></li>
+            <li><a href="https://www.facebook.com/MCHSOaggies/">facebook</a></li>
```
Comment (jswood23) [inline app/views/external/contact.html.erb]: Let's use `link_to` here instead of `<a>` so we can be more consistent with styling later on.

**Review by jswood23 — CHANGES_REQUESTED** [review]

**Comment by NR229** [comment]
Retry with different branch

**Pull request closed without merging**